src/main/java/edu/kis/powp/jobs2d/drivers/DriverFeaturesManager.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.shape.line.BasicLine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.observer.Publisher;
+
+public class DriverFeaturesManager extends DriverManager {
+    private DriversComposite currentDriver = new DriversComposite();
+    private final Publisher changePublisher = new Publisher();
+    private final static Job2dDriver PREVIEW_DRIVER = new LineDriverAdapter(null, new BasicLine(),"preview");
```

Comment:
Why PREVIEW? There is no 'preview' feature in your task.

Reply:
Unfortunately, DriverManger with DriverComposite inside, which contains Job2dDriver named preview doesn't work. We can make it work removing it.